src/components/testCaseList/TestCaseList.tsx

```diff
@@ -50,24 +86,19 @@ const TestCaseList = () => {
               </thead>
               <tbody>
                 {testCases?.map((testCase) => (
-                  <tr tw="border-b" key={testCase.id}>
-                    <TD>{testCase.title}</TD>
-                    <TD>{testCase.series}</TD>
-                    <TD>NA</TD>
-                    <TD>
-                      <Button
-                        onClick={() => {
-                          navigate(`./${testCase.id}`);
-                        }}
-                        data-testid={`edit-test-case-${testCase.id}`}
-                      >
-                        Edit
-                      </Button>
-                    </TD>
-                  </tr>
+                  <TestCaseComponent
+                    testCase={testCase}
+                    children={null}
+                    setActiveItem={setActiveItem}
+                    activeItem={activeItem}
+                    index={testCase.id}
```

Comment:
It is good to have a unique `key` for each item while iterating. Maybe renaming the `index` to `key` would work.

Reply:
@adongare Done.